core/src/main/java/org/geysermc/geyser/translator/text/MessageTranslator.java

```diff
@@ -320,7 +320,7 @@ public static void handleChatPacket(GeyserSession session, Component message, in
             if (parameters.contains(TextDecoration.Parameter.CONTENT)) {
                 args.add(message);
             }
-            withDecoration.args(args);
+            withDecoration.arguments(args);
```

Comment:
Velocity uses Adventure 4.15 now, right? That would be the only concern.

Reply:
`.args()` was introduced in 4.0.0, and marked deprecated since 4.15.0 with the introduction of the `.arguments()` method. Velocity does use 4.15.0 by now, but it looks like we still use 4.14.0 at the moment? Not sure why that is, but ill deal with this in a different PR. reverted for now